app/src/main/java/org/fossasia/pslab/activity/OscilloscopeActivity.java

```diff
@@ -426,15 +426,19 @@ protected Void doInBackground(String... params) {
                 analogInput = params[0];
                 //no. of samples and timegap still need to be determined
                 scienceLab.captureTraces(1, 1000, 10, analogInput, false, null);
+                try {
+                    Thread.sleep((long) (1000 * 1000 * 10 * 1e-6));
```

Comment:
Yes. Its just for testing to be clear of (samples * timeGap * 1e-3). Later it would be replaced with variables.